src/ui/server/constants/field-ids/insurance/index.ts

```diff
@@ -19,6 +19,7 @@ const INSURANCE_FIELD_IDS = {
     COVER_PERIOD_ID: 'coverPeriodId',
     HAS_END_BUYER: 'hasEndBuyer',
     HAVE_AN_ACCOUNT: 'haveAnAccount',
+    HAS_REVIEWED_ELIGIBILITY: 'hasReviewedEligibility',
```

Comment:
Can we make sure this is added to other field ID constants - I know this is only for the UI but when we get a single, shared package for constants, having this missing could trip us up. What do you think